### PR TITLE
Expand user documentation and spruce some things up

### PR DESCRIPTION
**Pull request by jacoballen1**

### CODE REVIEW

File: docs/users/configuration.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+Before we continue, it is useful to take a step back and note that Velocity uses the [TOML](https://github.com/toml-lang/toml) format for its
+configuration. TOML was designed to be easy to understand, so you should not have difficulty understanding
+Velocity's configuration file.
+
+##Root section
+
+These settings mostly cover the basic, most essential settings of the proxy
+
+<style>
```
Comment (astei) [inline docs/users/configuration.md]: These need to be global, per-theme styles

File: docs/users/configuration.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+}
+.secondary-table th,
+.secondary-table td {
+  text-align: left;
+  width: 4px;
+  padding: .5rem;
+}
+</style>
+
+<div class="ox-hugo-table main-table">
```
Comment (astei) [inline docs/users/configuration.md]: Don't clutter with these `div`s

File: docs/users/configuration.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+| `id`          	| UUID    	| Randomly generated UUID 	| A randomly generated UUID that uniquely identifies your Velocity server. You should not alter this setting. 	|
+| `log-failure` 	| Boolean 	| `false`                 	| Whether or not Velocity should log whenever it fails to connect to bStats.                                  	|
+
+</div>
+
+##The default configuration
+
+Below is the default configuration file for Velocity, `velocity.toml`.
+
+```plain
```
Comment (astei) [inline docs/users/configuration.md]: Use toml for the language here

File: docs/users/forwarding.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+## Configuring modern forwarding
+`modern` forwarding is a Velocity-native format. It forwards all player information in an efficient binary format and
+employs a MAC code to make it much more difficult to trick the server into impersonating your Velocity proxy.
+However, it is only available for Minecraft 1.13 or higher.
+
+To use modern forwarding with any supported server implementation, set the `player-info-forwarding` setting in
+`velocity.toml` to `modern`. You must also change the `forwarding-secret` setting to a unique secret. You then need
+to ensure your server is properly configured to use Velocity forwarding. 
+
+```css
```
Comment (astei) [inline docs/users/forwarding.md]: Can we do better here?

**Review by astei — CHANGES_REQUESTED** [review]

File: docs/users/configuration.md (hunk 1 above)
Comment (jacoballen1) [reply docs/users/configuration.md]: fixed in 4ce76bcd44a19018a6228bc7df433f6eba042248

File: docs/users/configuration.md (hunk 2 above)
Comment (jacoballen1) [reply docs/users/configuration.md]: fixed in 4ce76bcd44a19018a6228bc7df433f6eba042248

File: docs/users/configuration.md (hunk 3 above)
Comment (jacoballen1) [reply docs/users/configuration.md]: fixed in 6605a4c52fe0d591205b8bd0268e19637160cde1

File: docs/users/forwarding.md (hunk 4 above)
Comment (jacoballen1) [reply docs/users/forwarding.md]: I have changed this in d6ec3c6779f859bfc809b0dbb3930a7da0edf052, let me know if you'd like another revision.

File: docs/users/getting-started.md [hunk 5]
```diff
@@ -84,4 +84,28 @@ Go ahead and type `end` at the console and press enter. The proxy shuts down:
 
 ### Configuring your servers
 
-We now need to configure each server to accept connections from the proxy.
\ No newline at end of file
+We now need to configure each server to accept connections from the proxy.
+
+Open up `velocity.toml` and navigate to the `[servers]` section. 
+This section of the config is very crucial as is what allows connection to your backend servers.
```
Comment (astei) [inline docs/users/getting-started.md]: This is very awkwardly worded. Perhaps we should rewrite this section?

File: docs/users/forwarding.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+Velocity.
+
+## Configuring Spigot / Paper
+
+To make Spigot or Paper understand the data forwarded from Velocity, set `settings.bungeecord` to `true` in your
+ `spigot.yml` and then reboot your server.
+ 
+## Configuring Sponge
+
+To configure Sponge to understand the data forwarded from Velocity, set `modules.bungecord` to `true` and
```
Comment (astei) [inline docs/users/forwarding.md]: Typo: `bungeecord`

File: docs/users/forwarding.md (hunk 4 above)
Comment (astei) [reply docs/users/forwarding.md]: The wording is improved somewhat, but there has to be a better way to display this warning.

File: docs/users/forwarding.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+# Configuring player information forwarding
+
+Velocity supports forwarding information about your players to your servers, such as IP addresses, UUIDs and skins.
```
Comment (astei) [inline docs/users/forwarding.md:3]: `Velocity supports forwarding player information such as IP addresses, UUIDs and skins to your servers. This is also known as IP forwarding.`

Honestly, this needs some serious revision (why should people configure modern forwarding?)

File: docs/users/configuration.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+# - "none":   No forwarding will be done. All players will appear to be connecting from the
+#             proxy and will have offline-mode UUIDs.
+# - "legacy": Forward player IPs and UUIDs in a BungeeCord-compatible format. Use this if
+#             you run servers using Minecraft 1.12 or lower.
+# - "modern": Forward player IPs and UUIDs as part of the login process using Velocity's
+#             native forwarding. Only applicable for Minecraft 1.13 or higher.
+player-info-forwarding-mode = "NONE"
+
+# If you are using modern IP forwarding, configure an unique secret here.
+forwarding-secret = "5L7eb15i6yie"
```
Comment (astei) [inline docs/users/configuration.md]: Probably a good idea _NOT_ to include this (or at least put in an obvious dummy string here)

File: src/components/navbar.js [hunk 9]
```diff
@@ -25,20 +24,28 @@ const NavbarItem = styled.li`
   display: inline;
   float: right;
   padding: 18px 10px;
-  
+   background: linear-gradient(to right, #0288d1, #0288d1);
```
Comment (astei) [inline src/components/navbar.js]: Be a bit more consistent with the spacing

File: src/components/full-width-button.js [hunk 10]
```diff
@@ -1,12 +1,16 @@
...
-  background-color: ${theme.colors.jumbotron};
-  padding: 1rem;
-  width: 100%;
-  margin-bottom: 2rem;
+  padding: .75em;
+  background: linear-gradient(to right, #0288d1, #0288d1);
+  background-repeat: no-repeat;
+  background-size: 0 100%;
+  transition: background-size 1s 0s;
+  &:hover{
```
Comment (astei) [inline src/components/full-width-button.js:12]: Same idea: let's indent this

**Review by astei — CHANGES_REQUESTED** [review]

File: docs/users/configuration.md (hunk 8 above)
Comment (jacoballen1) [reply docs/users/configuration.md]: This iteration of the config was copied from the existing docs. However I can still change it

File: docs/users/forwarding.md (hunk 7 above)
Comment (jacoballen1) [reply docs/users/forwarding.md:3]: Would it be better to just leave that to line 7? As it explicitly talks about modern forwarding and is directly below?

File: src/components/layout.js [hunk 11]
```diff
@@ -25,13 +25,29 @@ export default function Layout({ jumbotron, children }) {
...
         }
+        
         td {
-         text-align: left;
-         padding: .5rem;
+          text-align: left;
+          padding: .5rem;
+        }
+        
+        .caution-header {
```
Comment (astei) [inline src/components/layout.js:38]: I'll likely want to rework how this works (using MDX and a React component for this), but this should be good for now.

File: src/components/layout.js [hunk 12]
```diff
@@ -25,13 +25,29 @@ export default function Layout({ jumbotron, children }) {
...
+          font-weight: bold;
+          background: rgb(184, 134, 11);
```
Comment (astei) [inline src/components/layout.js:40]: Need to put this in the theme

**Review by astei — APPROVED** [review]

**Pull request merged**